### PR TITLE
Several New Moon Events

### PR DESCRIPTION
**Pull request by redhairedl**
New Moon Events

### CODE REVIEW

File: resources/dicts/events/death/general/elder.json [hunk 1]
```diff
@@ -130,5 +130,14 @@
         "cat_skill": null,
         "other_cat_trait": null,
         "other_cat_skill": null
+    },
+    {
+        "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "other_cat", "other_cat_mate", "multi_death", "old_age", "all_lives"],
+        "camp": "Any",
+        "death_text": "m_c and r_c loved each other so much, they could not be parted, even by death. c_n finds their bodies, still wrapped around each other in the morning.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/elder.json]: you can remove the first comma and third comma.

File: resources/dicts/events/death/general/elder.json [hunk 2]
```diff
@@ -130,5 +130,14 @@
...
+        "history_text": {
+            "reg_death": "This cat died from old age alongside the cat {PRONOUN/m_c/subject} {VERB/love/loves}.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/elder.json]: Since this is a a multi-cat event, pronoun tags work weirdly.  basically, both cats will get this event, so at least one cat will have the wrong pronoun.  try to find a way to reword this to be pronoun-less

File: resources/dicts/events/death/general/general.json [hunk 3]
```diff
@@ -300,5 +300,28 @@
         "cat_skill": null,
         "other_cat_trait": null,
         "other_cat_skill": null
+    },
+    {
+        "tags": ["other_cat", "Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare"],
+        "camp": "Any",
+        "death_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins, when suddenly {PRONOUN/m_c/subject} {VERB/is/are} ambushed and killed by a badger. c_n finds {PRONOUN/m_c/poss} body after a long search and wonders why {PRONOUN/m_c/subject} decided to leave the safety of the c_n.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/general.json]: I do agree with the comment that was made in the thread that the badger feels out of place.  since the event spends so much time dwelling on the conflict of m_c not knowing where they came from, it feels weird to suddenly switch to a second important event.  They're fighting for attention too much, which isn't ideal for a short event.  I would suggest removing the specific death reason (i.e. badger) and instead just have m_c's body be found after the search.  perhaps something like "When it's realized that m_c hasn't been seen all day, a search party is sent out.  Only to find the still, cold body of m_c.  c_n is left to wonder what killed m_c and why they left camp in the first place."

File: resources/dicts/events/death/general/general.json [hunk 4]
```diff
@@ -300,5 +300,28 @@
...
+        "history_text": {
+            "reg_death": "Curiosity killed this cat.",
+            "lead_death": "lost a life because of curiosity."
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/general.json:310]: i do adore the curiosity killed the cat reference lmao, very clever of you

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

File: resources/dicts/events/death/general/elder.json [hunk 5]
```diff
@@ -134,10 +134,10 @@
     {
         "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "other_cat", "other_cat_mate", "multi_death", "old_age", "all_lives"],
         "camp": "Any",
-        "death_text": "m_c and r_c loved each other so much, they could not be parted, even by death. c_n finds their bodies, still wrapped around each other in the morning.",
+        "death_text": "m_c and r_c loved each other so much they could not be parted, even by death. c_n finds their bodies still wrapped around each other in the morning.",
         "history_text": {
-            "reg_death": "This cat died from old age alongside the cat {PRONOUN/m_c/subject} {VERB/love/loves}.",
-            "lead_death": "died alongside the cat {PRONOUN/m_c/subject} {VERB/love/loves}."
+            "reg_death": "This cat died from old age curled around the love of their life.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/elder.json]: "their" is still a pronoun

File: resources/dicts/events/death/general/general.json [hunk 6]
```diff
@@ -304,7 +304,7 @@
...
         "camp": "Any",
-        "death_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins, when suddenly {PRONOUN/m_c/subject} {VERB/is/are} ambushed and killed by a badger. c_n finds {PRONOUN/m_c/poss} body after a long search and wonders why {PRONOUN/m_c/subject} decided to leave the safety of the c_n.",
+        "death_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins. Frantic with worry and confusion after finding m_c's empty nest in the morning, they launch a search. c_n finds {PRONOUN/m_c/poss} cooling body after a long search and wonders why {PRONOUN/m_c/subject} decided to leave the safety of the c_n, and what killed {PRONOUN/m_c/object}.",
         "history_text": {
             "reg_death": "Curiosity killed this cat.",
             "lead_death": "lost a life because of curiosity."
```
Comment (scribblecrumb) [inline resources/dicts/events/death/general/general.json:310]: forgot to mention this before, but if you want a leader to be able to get this then you'll have to copy paste it into the leader.json as well.  leaders don't have access to the general death events.  This also applies to the elder old age event.

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Review by Ryos00 — APPROVED** [review]
